### ISSUE TITLE
In 10.11, request() doesn't properly handle HTTPS

### ISSUE THREAD

**Issue by Aghassi** [Alamofire/Alamofire#799]
In OS X 10.11 GM, `Alamofire.request()` will not handle an `HTTPS` call properly, causing a nil return since 10.11 doesn't process http calls by default in a project (although an exception can be made in the plist file). Exact error:

```
App Transport Security has blocked a cleartext HTTP (http://) resource load since it is insecure. Temporary exceptions can be configured via your app's Info.plist file.
fatal error: unexpectedly found nil while unwrapping an Optional value
```

**Comment by tobiasoleary** [comment]
What's the URL? The error message says your not using HTTPS but HTTP.

**Comment by Aghassi** [comment]
` https://itunes.apple.com/lookup?id=`

That's the one. Which is why I posted. Sorry forgot it in the original post.

**Comment by cnoon** [comment]
Could you provide more info to go on?

* Full sample code showing how you're making the call
* Whether this is an iOS or OS X app
* What version of Alamofire you are using
* How you installed Alamofire into your project

This is the exact error that you get from ATS when you try to call a service over `http`.

**Comment by Aghassi** [comment]
@cnoon 

The code block in question is on line 50 here: https://github.com/Aghassi/Notify/blob/master/Notify/ITunesNotificationHandler.swift

This is an OS X application. 

We are using this version of Alamofire from 4 days ago: https://github.com/Alamofire/Alamofire/tree/71a26d641d3033540ef12fa632222b6b285e7b2e

We installed Alamofire into our project using git submodules.

**Comment by cnoon** [comment]
Thanks @Aghassi. This will take me a bit to figure out what's going on here. I haven't put together OS X app yet with Alamofire. 

Does anyone else in the community that works on OS X regularly have any idea what @Aghassi could be running into?

**Comment by kiancheong** [comment]
I did a fork of your project and ran it on my Mac. Everything seemed fine. Do you have a proxy running?
![screen shot 2015-09-23 at 4 34 11 pm](https://cloud.githubusercontent.com/assets/153908/10040760/13dde4de-6211-11e5-8d1e-b22e8e4d2c79.png)

**Comment by Aghassi** [comment]
@kiancheong Right now it works fine because we disabled App Transport Security, see here: https://forums.developer.apple.com/thread/3544

Try enabling it and see if it still works on 10.11

**Comment by kiancheong** [comment]
I just did another test with both ATS enabled and disabled, both times they ran okay. I'm on 10.11 Beta (15A279b) if that helps.

**Comment by Aghassi** [comment]
This is so strange. I'm running 15A282b of 10.11 with Xcode 7 GM. I did some more testing with random songs, and only once could I get it to say that with iTunes (with ATS enabled). When I was trying it earlier it was happening every time. Maybe we had a mistake on our end. Also, I couldn't reproduce the time it just happened because when I played the same song it seemed to work fine. If I run into this again I'll reopen this, but for right now I'll close the thread. I'm still baffled.
